api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -1054,6 +1072,306 @@ public void findSurveyVersionByQuestionConceptId() {
     jdbcTemplate.execute("drop table cb_survey_attribute");
   }
 
+  @Disabled(
+      "Cannot be tested without SQL/Java modifications for H2. "
+          + "(a) Native query and not JPQL and (b) MATCH ... against... not H2 function")
+  public void findSurveyCount() {}
+
+  @Test
+  public void findSurveyVersionByQuestionConceptIdAndAnswerConceptId() {
+    jdbcTemplate.execute(
+        "create table cb_survey_version(survey_version_concept_id integer, survey_concept_id integer, display_name varchar(50), display_order integer)");
+    jdbcTemplate.execute(
+        "create table cb_survey_attribute(id integer, question_concept_id integer, answer_concept_id integer, survey_version_concept_id integer, item_count integer)");
+    jdbcTemplate.execute(
+        "insert into cb_survey_version(survey_version_concept_id, survey_concept_id, display_name, display_order) values (999, 111, 'May 2020', 1)");
+    jdbcTemplate.execute(
+        "insert into cb_survey_attribute(id, question_concept_id, answer_concept_id, survey_version_concept_id, item_count) values (1, 222, 333, 999, 200)");
+
+    List<SurveyVersion> response =
+        controller
+            .findSurveyVersionByQuestionConceptIdAndAnswerConceptId(
+                WORKSPACE_NAMESPACE, WORKSPACE_ID, 111l, 222l, 333l)
+            .getBody()
+            .getItems();
+    assertThat(response.get(0).getSurveyVersionConceptId()).isEqualTo(999);
+    assertThat(response.get(0).getDisplayName()).isEqualTo("May 2020");
+    assertThat(response.get(0).getItemCount()).isEqualTo(200);
+
+    jdbcTemplate.execute("drop table cb_survey_version");
+    jdbcTemplate.execute("drop table cb_survey_attribute");
+  }
+
+  @Disabled(
+      "Cannot be tested without SQL/Java modifications for H2. "
+          + "(a) Native query and not JPQL and (b) MATCH ... against... not H2 function")
+  public void findDrugIngredientByConceptId() {}
+
+  @Test
+  public void findAgeTypeCounts() throws Exception {
+    Calendar today = Calendar.getInstance();
+    int age = 25;
+    // birthday-month-date today
+    DbPerson todayDob =
+        DbPerson.builder()
+            .addPersonId(1000L)
+            .addDob(Date.valueOf(getDateRelativeToToday(-age, 0, 0)))
```

Comment:
```suggestion
            .addDob(getDateRelativeToToday(-age, 0, 0))
```
Please do the conversion to java.sql.Date in the extracted method.